### DISCUSSION TITLE
Fullset of Hardware incl. Case/box

### DISCUSSION THREAD

**Discussion by thomasklein1975** [Yogui79/IntexPureSpa#35] [General]
Hi,

would it be possible to get complete needed hardware incl. case/box assembled for this project?

Greetings
Thomas

**Comment by LEECHER1** [comment]
Hey,

Sorry, it is for us to much work.
It is only a Tutorial to build it. 

Best regards

**Reply by thomasklein1975** [reply]
ok thx. can i use nano instead of esp32 with LC12S for full functionality?

**Comment by thomasklein1975** [comment]
where can i buy hardware without waiting so long time especially for lc12s?
